sdk/include/opentelemetry/sdk/instrumentationscope/instrumentation_scope.h

```diff
@@ -27,14 +30,66 @@ class InstrumentationScope
    * @param name name of the instrumentation scope.
    * @param version version of the instrumentation scope.
    * @param schema_url schema url of the telemetry emitted by the library.
+   * @param attributes attributes of the instrumentation scope.
    * @returns the newly created InstrumentationScope.
    */
-  static nostd::unique_ptr<InstrumentationScope> Create(nostd::string_view name,
-                                                        nostd::string_view version    = "",
-                                                        nostd::string_view schema_url = "")
+  static nostd::unique_ptr<InstrumentationScope> Create(
+      nostd::string_view name,
+      nostd::string_view version    = "",
+      nostd::string_view schema_url = "",
+      ::std::initializer_list<
+          ::std::pair<nostd::string_view, opentelemetry::common::AttributeValue>> attributes = {})
```

Comment:
`InstrumentationScope` is part of SDK.  Can we just use `opentelemetry::sdk::common::AttributeMap` for attributes, similar to how we use inside Resources.

Reply:
Thanks and fixed.